source/sass/patterns/molecules/figure.scss

```diff
@@ -0,0 +1,32 @@
+@import "../../variables/breakpoint";
+@import "../../mixins/spacing";
+@import "../../mixins/typography";
+
+.figure {
+  @include block-spacing($end: $baselinegrid-space-medium);
+}
+
+.figure__image {
+  @include block-spacing($start: $baselinegrid-space-extra_small, $end: $baselinegrid-space-extra_small);
+  @include margin(12px, inline);
+  border: 1px solid $color-text-dividers;
+  display: block;
+
+}
+
+.figure__heading {
+  @include caption-typography();
+}
+
+.figure__caption {
+  @include caption-typography();
+  @include margin(12px, inline);
```

Comment:
Need to work how this value is calculated.

Reply:
This matches the baseline grid extra-small value in the block axis, but it shouldn't scale with the baseline grid, so we shouldn't impose an equality for that onto the inline axis. A rare occasion of a magic number. Or at least one with no currently identified relationship to anything else.

Reply:
Another use for the variable mentioned in https://github.com/libero/pattern-library/pull/87#discussion_r281663788.

Reply:
Due to https://github.com/libero/pattern-library/pull/87/files#diff-cc0339e6b177e5f6b7984fa34d9bb124R14, this margin declaration can be replaced by:
```
  @include margin($image-inline-spacing, inline);
```